omero-deploy/Dockerfile

```diff
@@ -12,7 +12,10 @@ RUN apt-get update -y && \
     sudo \
     vim
 
-RUN /opt/anaconda/bin/pip install omego
+RUN /opt/anaconda/bin/pip install \
+    omego \
+    "django<1.9" \
```

Comment:
Could do, but since this is a test-image it'll have to be done at runtime after installing the server.